docs/Serverless.md

```diff
@@ -0,0 +1,86 @@
+<h1 align="center">Serverless</h1>
+
+> Fastify is not designed to run on the AWS Lambda platform. The Fastify framework is designed to make implementing a traditional HTTP/S server easy. The AWS Lambda platform services requests in a completely different manner than a traditional HTTP/S server and is therefore at odds with Fastify's whole design. Regardless, it is possible to use Fastify in this environment if you so choose. This document describes how to do it. However, remember that this is an unsupported usage of Fastify and you are on your own in terms of support.
+
+
+Run serverless applications and REST APIs using your existing Fastify application.
```

Comment:
I think this should go before the warning.

Reply:
√